source/common/http/http1/codec_impl.cc

```diff
@@ -314,47 +314,13 @@ void RequestStreamEncoderImpl::encodeHeaders(const HeaderMap& headers, bool end_
   StreamEncoderImpl::encodeHeaders(headers, end_stream);
 }
 
-http_parser_settings ConnectionImpl::settings_{
```

Comment:
While you're doing your clean-up, WDYD of landing some of the pure refactor as its own PR?
It'd let us land some of this ASAP, make for easier and smaller reviews, etc.

Reply:
Sure I can break this up a bit (I was admittedly surprised how large the delta was once I got the refactor building)